### ISSUE TITLE
Always Roll 1 and 10 Out of Combat Options

### ISSUE THREAD

**Issue by SunsetteSkye** [cabarius/ToyBox#316]
Similar to the 'Always Roll 20 Out of Combat' Option -- this would make stealth a much more viable strategy without inadvertently boosting other things ridiculously high like dialogue options.

**Comment by Amariithynar** [comment]
Agreed! I currently have "roll 20 out of combat" simply because there is no "always take 10 out of combat" option, which I sorely desire. Thankfully the way I've built my guys I never need to worry about the extra 10 that it gives over rolling with just 20s for general issues, at least, but as it is currently does mean you can get a free critical hit every time you open with an attack from out of their detection range to initiate combat. They'd have to make it so that if a 10 can't get it, then it goes up to a 20 if it's not dangerous, though (how many parties do you know that don't take 20 to search an area for any hidden little things? We're all packrats here)!
